### PR TITLE
mention mysql replication 

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Chinese part of #5137 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5872
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-compatibility.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ TiDB is fully compatible with the MySQL 5.7 protocol and the common features and
 
 However, some features of MySQL are not supported. This could be because there is now a better way to solve the problem (such as XML functions superseded by JSON), or a lack of current demand versus effort required (such as stored procedures and functions). Some features might also be difficult to implement as a distributed system.
 
+- In addition, TiDB does not support the MySQL replication protocol, but provides specific tools to replicate data with MySQL.
+    - Replicate data from MySQL: [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/stable/overview) is a tool for migrating data from MySQL/MariaDB into TiDB and replicating the incremental data.
+    - Replicate data to MySQL: [TiCDC](/ticdc/ticdc-overview.md) is a tool for replicating the incremental data of TiDB and implemented by pulling TiKV change logs. You can use [MySQL sink](/ticdc/ticdc-overview.md#sink-support) to replicate the incremental data of TiDB to MySQL.
```
Comment (TomShawn) [inline mysql-compatibility.md:15]:
```suggestion
    - Replicate data to MySQL: [TiCDC](/ticdc/ticdc-overview.md) is a tool for replicating the incremental data of TiDB, which is implemented by pulling TiKV change logs. TiCDC uses the [MySQL sink](/ticdc/ticdc-overview.md#sink-support) to replicate the incremental data of TiDB to MySQL.
```

**Comment by TomShawn** [comment]
rest /lgtm

**Comment by TomShawn** [comment]
rest 
/lgtm

File: mysql-compatibility.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ TiDB is fully compatible with the MySQL 5.7 protocol and the common features and
 
 However, some features of MySQL are not supported. This could be because there is now a better way to solve the problem (such as XML functions superseded by JSON), or a lack of current demand versus effort required (such as stored procedures and functions). Some features might also be difficult to implement as a distributed system.
 
+- In addition, TiDB does not support the MySQL replication protocol, but provides specific tools to replicate data with MySQL.
+    - Replicate data from MySQL: [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/stable/overview) is a tool for migrating data from MySQL/MariaDB into TiDB and replicating the incremental data.
```
Comment (lance6716) [inline mysql-compatibility.md:14]:
```suggestion
    - Replicate data from MySQL: [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/stable/overview) is a tool for migrating data from MySQL/MariaDB into TiDB and supports replicating the incremental data.
```

seems strange if `migrating data` and `replicating the incremental data` are in same level

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by lance6716 — APPROVED** [review]

**Pull request merged**